README.md

```diff
@@ -4,22 +4,38 @@ This code simulates proton-electron collisions using Pythia8 and saves in HepMC
 
 ## Installation
 
-To run this code, you need to install [Pythia8](https://pythia.org/), a program for the generation of high-energy physics collision events. You can download Pythia8 using:
+1) Get and run [eic-shell](https://eic.github.io/tutorial-setting-up-environment/02-eic-shell/index.html)
+
+2) Clone and make Pythia8.306
 ```
-git clone git@gitlab.com:Pythia8/releases.git --branch pythia8XXX --single-branch
+mkdir Pythia8306
+cd Pythia8306
+git clone https://gitlab.com/Pythia8/releases.git --branch pythia8306 --single-branch
+cd releases
+./configure --with-hepmc3
+make
+cd bin
+export LD_LIBRARY_PATH=$(pythia8-config --libdir):${LD_LIBRARY_PATH}
+cd ../../
 ```
-where XXX is the PYTHIA version number. You can find the list of Pythia8 releases [here](https://pythia.org/releases/). For this code version 8.306 was used. 
-This will download zip `pythia8306.tgz`. Unzip and expand it with:
+
+3) Clone and compile LambdaGen against Pythia
 ```
- tar xvfz pythia8306.tgz.
+git clone https://gitlab.com/eic/LambdaGen.git --branch pythia8.306-1.0 --single-branch
+cd LambdaGen
+g++ main41_Lam.cc -o main41_Lam `pythia8-config --cflags --libs --hepmc3`
```

Comment:
Technicaly it's c++ and this may make a difference if c++ standard is added.
```suggestion
g++ main41_Lam.cc -o main41_Lam `pythia8-config --cxxflags --libs --hepmc3`
```